package.json

```diff
@@ -5,17 +5,22 @@
   "description": "",
   "keywords": [],
   "dependencies": {
+    "assets-webpack-plugin": "^3.5.1",
     "aws-sdk": "^2.1.36",
+    "babel-loader": "^7.0.0",
+    "babel-polyfill": "^6.16.0",
     "body-parser": "^1.16.1",
     "cfenv": "^1.0.0",
     "connect-session-sequelize": "^4.1.0",
     "core-js": "^2.4.1",
     "db-migrate": "^0.9.17",
     "deep-extend": "^0.4.1",
+    "ejs": "^2.5.6",
```

Comment:
Also not sure about having this in the dependencies since we only seem to be using it to js and css bundles in `index.html`.